content/en/platform/corda/5.0-beta/operating/operating-tutorials/configuring-manual-approval-rules.md

```diff
@@ -0,0 +1,49 @@
+---
+date: '2023-03-15'
+title: "Configuring Manual Approval Rules"
+menu:
+  corda-5-beta:
+    identifier: corda-5-beta-config-manual-approval
+    parent: corda-5-beta-tutorials-reg-requests
+    weight: 1000
+section_menu: corda-5-beta
+---
+As described in [Manual Registration Approval]({{< relref "../manual-registration.md" >}}), membership groups can require that member registration requests are manually approved (or declined). This section describes how to manage manual registration rules.
+
+## Adding a Group Approval Rule
+
+To add a group approval rule, use the [PUT method of the mgm/{holdingidentityshorthash}/approval/rules endpoint](../../rest-api/C5_OpenAPI.html#tag/MGM-API/operation/post_mgm__holdingidentityshorthash__approval_rules) of the REST API.
+
+For example, to specify that all requests in the `corda` namespace must be manually approved:
```

Comment:
@YashNabar This wasn't very clear in your draft. Can you be sure to review? Thanks

Reply:
Perhaps something along the lines of
```suggestion
For example, to specify that all requests with changes to keys in the 'corda' namespace must be manually approved:
```